addons/l10n_co_pos_report/models/report_sale_details.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, models
+
+
+class PosOrder(models.AbstractModel):
+    _inherit = 'report.point_of_sale.report_saledetails'
+    _description = 'Point of Sale Details'
+
+    @api.model
+    def prepare_sale_details(self, orders, domain, date_start, date_stop, config_ids, session_ids, include_products):
+        result = super(PosOrder, self).prepare_sale_details(orders, domain, date_start, date_stop, config_ids, session_ids, include_products)
+        result['include_products'] = include_products
```

Comment:
If include_products is always True when Columbia, you can remove all signature changes, and the wizard stuff, and here you just do:

include_products = self.env.company.country_id.code == 'CO' and len(config_ids)
 == 1

but, I am not sure of using the 'self.env.company' in multi-company environement, and probably wha could be better is:

if len(config_ids) == 1:
    if config_ids.company_id.country_id.code == 'CO':
       result.update({
                'include_products': True,
                'pos_config': self.env['pos.config'].browse(config_ids),
                'first_ref': orders and orders[-1].name,
                'last_ref': orders and orders[0].name,
                'total_payment_count': sum(payment.get('count') for payment in result.get('payments')),
            })

Reply:
Hello @pimodoo 

we are including product details all the time.
but in the Colombian report, it is said to give the option to select whether the user wants to include the product in the report or not

Reply:
Why do you need to give the option, and not always include it when we are in the columbian report? More over, you cannot modify the signature of public functions in stable release, as it potentially break customisations of customers if they override the function

Reply:
ok, then I will make changes according to it